integration/jnifuse/jnifuse-java/src/main/java/alluxio/jnifuse/utils/Environment.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.jnifuse.utils;
+
+public class Environment {
+  private static String OS = System.getProperty("os.name").toLowerCase();
+  private static String ARCH = System.getProperty("os.arch").toLowerCase();
+
+  public static boolean isAarch64() {
+    return ARCH.contains("aarch64");
+  }
+
+  public static boolean isPowerPC() {
+    return ARCH.contains("ppc");
+  }
+
+  public static boolean isS390x() {
+    return ARCH.contains("s390x");
+  }
+
+  public static boolean isWindows() {
+    return (OS.contains("win"));
```

Comment:
unnecessary outer `()`, same comments for below `returns`

Reply:
Sure.